Packs/SophosCentral/ReleaseNotes/1_3_2.md

```diff
@@ -0,0 +1,4 @@
+#### Integrations
+##### Sophos Central
+- Added new argument to sophos-central-endpoint-list to query endpoints by IP hostname
```

Comment:
Command names should be sorounded with ***<command-name>***, as specified in https://xsoar.pan.dev/docs/documentation/release-notes.
I'll fix it in the last PR release notes as well (https://github.com/demisto/content/pull/35357).
```suggestion
- Added new argument to ***sophos-central-endpoint-list*** to query endpoints by IP hostname
```